scripts/ci-e2e.sh

```diff
@@ -59,10 +59,22 @@ source "${M3_DEV_ENV_PATH}/lib/ironic_basic_auth.sh"
 source "${M3_DEV_ENV_PATH}/lib/ironic_tls_setup.sh"
 
 # Parameterize e2e_config
-export CAPI_FROM_RELEASE="${CAPI_FROM_RELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v0.4.")}"
+CAPI_UPGRADE_FROM_RELEASE="${UPGRADE_FROM_RELEASE:-v0.5.}"
+if [[ "${UPGRADE_FROM_RELEASE}" == "v0.5." ]]; then
+    CAPI_UPGRADE_FROM_RELEASE="v0.4."
+fi
+export CAPI_FROM_RELEASE="${CAPI_FROM_RELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "${CAPI_UPGRADE_FROM_RELEASE}")}"
 export CAPI_TO_RELEASE="${CAPIRELEASE}"
-export CAPM3_FROM_RELEASE="${CAPM3_FROM_RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v0.5.")}"
-export CAPM3_TO_RELEASE="${CAPM3RELEASE}"
+case ${CAPI_FROM_RELEASE} in
+v0.4*)
+    export CONTRACT_FROM="v1alpha4"
+    ;;
+v1.*)
+    export CONTRACT_FROM="v1beta1"
+    ;;
```

Comment:
I may be missing something, but why add this here and do the templating magic for the e2e_conf.yaml instead of just putting it directly in the e2e_config.yaml?

Reply:
I don't get the benefit. It is much harder to understand what is tested and we still need to change this when there are new releases, right?

Reply:
I have no strong opinion on that we pushed to parameterize e2e config https://github.com/metal3-io/cluster-api-provider-metal3/pull/599 I do not want to revert it to static